app/models/display_ad.rb

```diff
@@ -30,6 +33,10 @@ def self.for_display(area)
     end
   end
 
+  def human_readable_placement_area
+    ALLOWED_PLACEMENT_AREAS_HUMAN_READABLE[ALLOWED_PLACEMENT_AREAS.find_index(placement_area)]
```

Comment:
Seeing this I wonder if we should disregard my above suggestion for the helper, remove the helper completely and make the display area a hash from the get-go and then just use `.keys` and `.values` where appropriate.

```ruby
ALLOWED_DISPLAY_AREAS = {
  sidebar_left: "Sidebar Left (First Position)",
  sidebar_left_2: "Sidebar Left (Second Position)",
  sidebar_right: "Sidebar Right"
}.with_indifferent_access.freeze
```

To generate the options for the `select` you can then simply use `Hash#invert`

```ruby
ALLOWED_DISPLAY_AREAS.invert
{
   "Sidebar Left (First Position)" => "sidebar_left",
  "Sidebar Left (Second Position)" => "sidebar_left_2",
                   "Sidebar Right" => "sidebar_right"
}
```

And this method would simply be

```ruby
ALLOWED_DISPLAY_AREA[placement_area]
```